transcom/mymove
# MB-14708 move closeout office data is included in customer app api response

## [Jira ticket](https://dp3.atlassian.net/browse/MB-14708) for this change

## Summary
A parameter called **closeout_office**  is  returned in the internal API response of the customer app.
<img width="824" alt="Screen Shot 2022-12-19 at 2 31 07 PM" src="https://user-images.githubusercontent.com/111006369/208505047-64b758da-16ac-4e16-abe5-0df3ec16f02d.png">

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```
</details>

## Verification Steps for Reviewers

Verify that the logged_in API response includes the closeout_office location:

Open developer tools network tab

Use this login for customer app: 2needcloseout@ppm.closeout (milmove)

Inspect network results for 'logged_in' api response

Check service_member > orders > moves in payload for 'closeout_location'

Note: Please be aware this object is null as test data is being generated in this upcoming ticket: [https://dp3.atlassian.net/browse/MB-14705](url)
